motoman_driver/src/industrial_robot_client/joint_trajectory_action.cpp

```diff
@@ -62,7 +63,11 @@ JointTrajectoryAction::JointTrajectoryAction() :
   // A parameter named topics_list contains the respective information for each of the groups
 
   std::string value;
-  ros::param::search("topics_list", value);
+  if(!ros::param::search("topics_list", value))
+  {
+    ROS_ERROR_STREAM(industrial_robot_client::motoman_utils::TOPIC_LIST_ERROR_MSG);
+    throw;
```

Comment:
Afaik, `throw` by itself is only allowed when re-`throw`ing an exception. That is not the case here, is it?